moment-element.html

```diff
@@ -33,80 +33,68 @@
          * For consistent results, parsing anything other than ISO 8601 strings
          * with the `inputFormat` property. More information in [moment String](http://momentjs.com/docs/#/parsing/string/).
          */
-        datetime: String,
+        datetime: {
+          type: String,
+          value: function() {
+            return new Date().getTime();
```

Comment:
The `new Date()` returns the date's time:

<img width="296" alt="screen shot 2016-04-26 at 21 24 21" src="https://cloud.githubusercontent.com/assets/1007051/14831279/47bcccb4-0bf5-11e6-915c-ffba4cecdf33.png">

And [moment docs](http://momentjs.com/docs/#/parsing/now/) says: `var now = moment()` is essentially the same as calling `moment(new Date())`

I think that `new Date()` is the best option.